src/snapshot-editor/src/info.rs

```diff
@@ -68,22 +74,36 @@ fn info_version(_: &MicrovmState, version: u16) -> Result<(), InfoVmStateError>
 }
 
 #[cfg(target_arch = "aarch64")]
+fn info_vcpu_states_arch(state: &VcpuState) {
+    println!("kvm_mp_state: {:#x}", state.mp_state.mp_state);
+    println!("mpidr: {:#x}", state.mpidr);
+    for reg in state.regs.iter() {
+        println!(
+            "{:#x} 0x{}",
+            reg.id,
+            reg.as_slice()
+                .iter()
+                .rev()
+                .map(|b| format!("{b:x}"))
+                .collect::<String>()
+        );
+    }
```

Comment:
I think it is better to move this to the `Debug` impl for `VcpuState`. This way we will not need 2 methods here + for `vm-state` command `VcpuState` will be nicely printed, otherwise `vcpu-state` and `vm-state` will print it in different ways.

Reply:
done.